shared/cm_filesystem.h

```diff
@@ -11,7 +11,7 @@
 #if defined(_WIN32)
 typedef struct _iobuf FILE;
 #else // !defined(_WIN32)
-typedef struct _IO_FILE FILE;
+#include <stdio.h>
```

Comment:
Was intentionally avoiding this include here, would rather we do:
```#elif defined(__APPLE__)```
```typedef struct [...?] FILE;```

And match the declaration in Apple's standard library.

Reply:
Whoops! Good point, thanks! Fixed.